cmd/thanos/receive.go

```diff
@@ -274,7 +274,7 @@ func runReceive(
 
 	level.Debug(logger).Log("msg", "setting up hashring")
 	{
-		if err := setupHashring(g, logger, reg, conf, hashringChangedChan, webHandler, statusProber, enableIngestion); err != nil {
+		if err := setupHashring(g, logger, reg, conf, hashringChangedChan, webHandler, statusProber, enableIngestion, conf.endpoint); err != nil {
```

Comment:
Is there a reason for passing the endpoint separately and not just taking it from `conf`?

Reply:
Good catch 👍